XmlFormatter/MainForm.cs

```diff
@@ -165,5 +151,112 @@ private void MainForm_DragDrop(object sender, DragEventArgs e)
             }
             TB_SelectedXml.Text = files[0];
         }
+
+        /// <summary>
+        /// This method is the click event for the about menu
+        /// </summary>
+        /// <param name="sender">The control sending the click event</param>
+        /// <param name="e">The event arguments</param>
+        private void MI_About_Click(object sender, EventArgs e)
+        {
+            Version version = GetApplicationVersion();
+            MessageBox.Show(GetStringVersion(version), "Version", MessageBoxButtons.OK, MessageBoxIcon.Information);
+        }
+
+        /// <summary>
+        /// This method will return you the current version of the application
+        /// </summary>
+        /// <returns>The current version of the application</returns>
+        private Version GetApplicationVersion()
+        {
+            Assembly assembly = Assembly.GetExecutingAssembly();
+            string versionString = "0.0.0";
+            using (Stream stream = assembly.GetManifestResourceStream("XmlFormatter.Version.txt"))
+            {
+                using (StreamReader reader = new StreamReader(stream))
+                {
+                    versionString = reader.ReadToEnd();
+                }
+            }
+
+            return ConvertInnerFormatToProperVersion(versionString);
+        }
+
+        /// <summary>
+        /// This method will convert a inner string version to a proper version instance
+        /// </summary>
+        /// <param name="stringVersion">The string version to use for converting</param>
+        /// <returns>A proper version class</returns>
+        private Version ConvertInnerFormatToProperVersion(string stringVersion)
+        {
+            stringVersion += ".0";
+            Regex regex = new Regex(@"[0-9]{1,}.[0-9]{1,}.[0-9]{1,}.[0-9]{1,}");
+            if (!regex.IsMatch(stringVersion))
+            {
+                stringVersion = "0.0.0.0";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Introduce a new variable instead of reusing the parameter 'stringVersion'.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=4882983)